independent-publisher-connectors/Airtable/apiDefinition.swagger.json

```diff
@@ -0,0 +1,420 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Airtable",
+    "description": "Custom connector to read and write data in Airtable. Airtable is a cloud based spreadsheet-like service.",
```

Comment:
@last72 Your connector will be certified and deployed to Production regions, So it will not be custom connector any more. Can you please remove `custom connector` word from the description and improve it with two or more sentences.

Documentation - [Description](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

Reply:
@diegobernardes Please ignore this comment, as your connector is IP connector.